R/geom-label-repel.R

```diff
@@ -351,13 +353,13 @@ makeContent.labelrepeltree <- function(x) {
           lineheight = row$lineheight
         ),
         rect.gp = gpar(
-          col = scales::alpha(row$colour, row$alpha),
-          fill = scales::alpha(row$fill, row$alpha),
+          col = scales::alpha(row$colour, row$label.col.alpha %||% row$alpha),
+          fill = scales::alpha(row$fill, row$label.fill.alpha %||% row$alpha),
           lwd = x$label.size * .pt
         ),
         segment.gp = gpar(
           col = scales::alpha(row$segment.colour %||% row$colour, row$segment.alpha %||% row$alpha),
-          lwd = row$segment.size * .pt,
+          lwd = (row$segment.size * .pt) %||% row$segment.linewidth,
```

Comment:
Since the default value for `segment.size` is 0.5, the value for `segment.linewidth` will not be used.

I'm willing to have another look at this suggestion, if you can help me to understand the intent of this change and how it should work for users. Backward compatibility is important to me, so I want to try to avoid breaking old code if possible.